### PR TITLE
[docs] Add status header for signalfx exporter

### PR DESCRIPTION
**Pull request by Frapschen**
**Description:** <Describe what has changed.>
Add status header for signalfx exporter

### CODE REVIEW

**Review by codeboten — COMMENTED** [review]
change looks ok, pinging code owners @dmitryax @pjanotti to verify stability status

File: exporter/signalfxexporter/README.md [hunk 1]
```diff
@@ -1,14 +1,21 @@
...
+[contrib]:https://github.com/open-telemetry/opentelemetry-collector-releases/tree/main/distributions/otelcol-contrib
+
 This exporter can be used to send metrics, events, and trace correlation to SignalFx.
 
 Apart from metrics, the exporter is also capable of sending metric metadata
 (properties and tags) to SignalFx. Currently, only metric metadata updates from
 the [k8s_cluster receiver](../../receiver/k8sclusterreceiver/README.md) are
 supported.
 
-Supported pipeline types: logs (events), metrics, traces (trace to metric correlation only)
```
Comment (dmitryax) [inline exporter/signalfxexporter/README.md:10]: Can you please move the whole sting `logs (events), metrics, traces (trace to metric correlation only)` to the table. This is pretty important information.

Comment (Frapschen) [reply exporter/signalfxexporter/README.md:10]: done

File: exporter/signalfxexporter/README.md [hunk 2]
```diff
@@ -1,14 +1,21 @@
...
 
+| Status                   |                                                                   |
+| ------------------------ |-------------------------------------------------------------------|
+| Stability                | [beta]                                                            |
+| Supported pipeline types | logs (events), metrics, traces (trace to metric correlation only) |
+| Distributions            | [contrib]                                                         |
+
+[beta]:https://github.com/open-telemetry/opentelemetry-collector#beta
+[contrib]:https://github.com/open-telemetry/opentelemetry-collector-releases/tree/main/distributions/otelcol-contrib
+
```
Comment (codeboten) [inline exporter/signalfxexporter/README.md:11]:
```suggestion
```

**Review by codeboten — COMMENTED** [review]
@dmitryax PTAL

**Review by dmitryax — APPROVED** [review]

**Pull request merged**